src/ansys/dpf/composites/layup_info/_layup_info.py

```diff
@@ -218,6 +218,9 @@ def get_dpf_material_id_by_analyis_ply_map(
                 layer_index = analysis_ply_info_provider.get_layer_index_by_element_id(
                     element_id_int
                 )
+                assert (
```

Comment:
Please check if this assertion makes sense.

Reply:
Yes that makes sense. I think it should never happen but the typechecker complained.